app/services/arcgis_api/mock/fixtures.rb

```diff
@@ -2,7 +2,7 @@ module ArcgisApi
   module Mock
     class Fixtures
       def self.request_suggestions_response
-        load_response_fixture('request_suggestions_response.json')
```

Comment:
Is the `request_suggestions_response.json` file unused now? If so, should it be removed?